web/src/lib/components/shared-components/focus-trap.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
```

Comment:
What is the rule of thumb to wrap other component with FocusTrap?

Reply:
I don't have a hard and fast rule, but it's something to consider any time you're making an element appear over the screen and have a `use:clickOutside` to dismiss it. The idea is that if there's an element on the screen that's preventing mouse users from interacting with elements behind it, keyboard users should also have the same experience.

Reply:
Thank you!